src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -76,7 +76,6 @@
                        Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
 
             <CustomControls:HotkeySettingsControl 
-                Width="240"
```

Comment:
I'd go for MinWidth instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%

Reply:
This scales fine.  Here is a screenshot from 200 DPI screen.
![image](https://user-images.githubusercontent.com/1462282/96911434-b36d6a00-1455-11eb-8d9f-7c4810544f47.png)